doc/reference/reports.rst

```diff
@@ -1,31 +1,101 @@
-.. _reference/reports:
-
 ============
 QWeb Reports
 ============
 
-Paper Formats
--------------
 
-There is a model called Paper Format allowing to define details specific to
-the PDF output.  These details include margins, header line, ... Everything
-related to the printed pdf.  Defining a paper format is not mandatory as there
-is a default one set on the company.  If you want a specific report to be
-associated to a specific paper format , just link the ir.actions.report.xml to
-it.
+Reports are written in HTML/QWeb, like all regular views in Odoo. You can use the usual QWeb control flow tools like t-if, t-foreach, etc. The PDF rendering is done by a tool outside the Odoo framework, **Wkhtmltopdf**.
+
+If you wish to create a report on a certain model, you will need to define this :ref:`report` and the :ref:`templates` it will use. If you wish, you can also specify a specific :ref:`paper_formats` for this report. Finally, if you wish to access more than your model, you can define a :ref:`custom_reports` class that gives you access to more models and records in the template.
+
+.. _report:
+
+Report 
+======
+
+Every report must be declared by an ir.actions.report.xml record. Once declared, the report will be automatically included in the "Print" menu in the form view of the relevent object, or when you select elements in a list view. In this record, you can specify:
+
+``name`` (mandatory)
+    only useful as a mnemonic/description of the report when looking for one in a list of some sort
+``model`` (mandatory)
+    the model your report will be about
+``report_type`` (mandatory)
+    either ``qweb-pdf`` for PDF reports or ``qweb-html`` for HTML
+``report_name``
+    the name of your report (which will be the name of the PDF output)
+``groups_id``
+    :class:`~openerp.fields.Many2many` field to the groups allowed to view/use the current report
+``paperformat_id``
+    :class:`~openerp.fields.Many2many` field to the paperformat you wish to use for this report (if not specified, the company format will be used)
```

Comment:
According to [the source](https://github.com/odoo/odoo/blob/1db27f417d20b92e6c390550dbe8365dfd7968f0/addons/report/models/report_paperformat.py#L155) this is an m2o field, not an m2m